### ISSUE TITLE
N Markers example is laggy with 2000 markers

### ISSUE THREAD

**Issue by hexadeciman** [realadvisor/rgm#8]
It would be awesome if there was a way to display a large number of markers without affecting performance, maybe using webgl or canvas would help. As soon as the number of markers makes too many dom elements it becomes laggy.

**Comment by istarkov** [comment]
Yep internally we had canvas drawer (on other map lib) it allowed to draw up to 4-5k markers. And used mix of React markers over to emulate hovers etc. Will add example when Ill have some free time.
